UserInterface/GameManager/GameManager.cs

```diff
@@ -54,7 +70,53 @@ public void RunApplication()
             _userInterface.DrawBorders(ConstantsRepository.BorderWidth, ConstantsRepository.BorderHeight);
             _userInterface.DrawBoard(_board);
             _window.SetCursorPosition(0, 20);
-            _userInterface.ShowMenuButtons();
+
+            foreach (var info in _gameItemsInfo)
+            {
+                _window.SetFontColor(info.Value.Color);
+                _userInterface.ShowMessage($"Press {info.Key} - to add {info.Value.Specie};");
+                _window.ResetFontColor();
+            }
+
+            Console.BackgroundColor = ConsoleColor.White;
+            _window.SetFontColor(ConsoleColor.Black);
+            _userInterface.ShowMessage(ConstantsRepository.ExitButtonDescription);
+            _window.ResetFontColor();
+
+            SetTimer();
+                        
+            bool isGameOnGoing = true;
+            while (isGameOnGoing)
+            {
+                ConsoleKey? consoleKey = _userInterface.GetInputKey();
+
+                _additionManager.ProcessAddNewItem(consoleKey, _gameItemsInfo, _gameItems, _board);
+
+                isGameOnGoing = (consoleKey != ConsoleKey.Q && consoleKey != ConsoleKey.Escape);
+            }
+
+            _timer.Stop();
+            _timer.Dispose();
+        }
+
+        /// <summary>
+        /// Sets timer.
+        /// </summary>
+        private void SetTimer()
+        {
+            _timer = new System.Timers.Timer(ConstantsRepository.TimeInterval);
+            _timer.Elapsed += OnTimedEvent;
```

Comment:
Here you can use shortened syntax for delegate.
`
_timer.Elapsed += (source, e) => RunGame();
`

Reply:
Done.